paperqa/settings.py

```diff
@@ -448,8 +448,9 @@ class AgentSettings(BaseModel):
     agent_prompt: str = (
         "Use the tools to answer the question: {question}\n\nThe {gen_answer_tool_name}"
         " tool output is visible to the user, so you do not need to restate the answer"
-        " and can simply terminate if the answer looks sufficient. The current status"
-        " of evidence/papers/cost is {status}"
+        " and can simply terminate if the answer looks sufficient. If you do not"
```

Comment:
I think this wording will confuse the agent (because it confuses me): "If you do not specify any tool calls, this means you failed to answer the question." -- I'm imagining our intended behavior is that the question seems impossible for the agent, right? Otherwise I think it should always keep trying. 

So maybe something like "If the answer does not look sufficient, and you have already tried to answer several times, you can terminate the question by specifying 0 tool calls."

Reply:
we'll need to test after this change, because the agent might be tempted to bail out too often.

Reply:
Okay, went with your suggestion here, thanks!